gm4_combat_expanded/data/gm4_combat_expanded/loot_tables/weapon/identification/rare/chaos.json

```diff
@@ -239,34 +239,32 @@
               {
                 "text": " "
               },
-              [
-                {
-                  "score": {
-                    "name": "$atkspeed_down_10",
-                    "objective": "gm4_ce_data"
-                  },
-                  "color": "red",
-                  "italic": false
-                },
-                {
-                  "text": ".",
-                  "color": "red",
-                  "italic": false
-                },
-                {
-                  "score": {
-                    "name": "$atkspeed_down_01",
-                    "objective": "gm4_ce_data"
-                  },
-                  "color": "red",
-                  "italic": false
-                }
-              ],
               {
-                "text": " "
-              },
-              {
-                "translate": "attribute.name.generic.attack_speed",
+                "translate": "attribute.modifier.equals.0",
+                "with": [
+                  [
+                    {
+                      "score": {
+                        "name": "$atkspeed_down_10",
+                        "objective": "gm4_ce_data"
+                      },
+                      "color": "red",
+                      "italic": false
```

Comment:
Here is a case where you may have weird coloring since it's only set for the first value before the decimal point. I'm not sure what ends up happening; the following text may be red or dark_green, but to have proper json, the color and italic should be set after line 262.